packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -334,6 +336,12 @@ const argv = yargs
       type: 'boolean',
       default: false,
     },
+    strictFlowTypes: {
+      describe:
+        'Generate flow types that more closely match the schema. This should be avoided ' +
+        'if the GraphQL server can change enum values without updating its schema',
```

Comment:
This is a bit confusing. Additional enum values is definitionally a schema update. Perhaps rephrase to talk about the case where enums add new values in the future

Reply:
What do you think of "Generate flow types that more closely match the schema. This means that you will have to update your application code whenever the GraphQL server adds new enum values. Not doing so could break your front-end."? Moreover, I'd like to explain a bit more the idea behind this, perhaps give an example, but I don't think that the `yargs` parameters description is the best case for that. I didn't see proper docs for the relay compiler, would a comment in this file would be ok for you?

Reply:
It would also be great to convey that this relates to the `%future added value` entry, as people may otherwise not immediately understand the relation and this being a great time to explain a bit more on that values purpose.

How about something like this?

> This option controls whether or not a catch-all entry is added to enum type definitions for values that may be added in the future.

Reply:
The last sentence is a bit confusing, this also applies to people doing server-side rendering.

Reply:
Your client? Your relay application? I'd like to emphasise that what I'm not speaking of the server, but the "client".

Reply:
Maybe something like this?

> This option controls whether or not a catch-all entry is added to enum type definitions for values that may be added in the future. Enabling this means you will have to update your application whenever the GraphQL server schema adds new enum values to prevent it from breaking.